src/core/storage/fileio/s3_api.cpp

```diff
@@ -410,7 +417,7 @@ list_objects_response list_objects_impl(s3url parsed_url, std::string proxy,
   }
 
   clientConfiguration.proxyHost = proxy.c_str();
-  clientConfiguration.requestTimeoutMs = 5 * 60000;
+  clientConfiguration.requestTimeoutMs = 10000;
```

Comment:
10s is enough for the list. There should be a global place to set all these configuration properties. Leave it for future people.

Reply:
well, it's recommended by blobby to set it to 5 mins.

Reply:
since we download by 64 MB, 5 mins are too many. let's reduce it to 2 mins.